packages/video_player/video_player/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.2.10
+* Support for iOS AVAudioSession Ambient mode which allows multiple
+  audio sources to play simultaneously and prevent background music
+  from stopping.
+* Uses '.setMixWithOthers(...)' on iOS to setup default AVAudioSession.
```

Comment:
Nit: "setup" is a noun; the verb is "set up".

Reply:
I fixed this